README.md

```diff
@@ -1,117 +1,38 @@
-gae-init
-========
+# vote4code
 
-[![Slack Status](https://gae-init-slack.herokuapp.com/badge.svg)](https://gae-init-slack.herokuapp.com)
+The vote4code website is place to upload two different versions of programming solutions to let the community vote for the best one. It's based on [**gae-init**](https://gae-init.appspot.com/), the easiest boilerplate to kick start new applications on Google
+App Engine using Python, Flask, RESTful, Bootstrap and tons of other [cool features](http://docs.gae-init.appspot.com/features/) out-of-the-box.
```

Comment:
I think it's more than enough to say that `It's based on [**gae-init**](https://gae-init.appspot.com).`  and that's it